lib/hsa/mcwamp_hsa.cpp

```diff
@@ -4264,16 +4282,13 @@ static Kalmar::hcCommandKind resolveMemcpyDirection(bool srcInDeviceMem, bool ds
 }
 
 inline hsa_status_t
-HSACopy::enqueueAsyncCopyCommand(Kalmar::HSAQueue* hsaQueue, const Kalmar::HSADevice *copyDevice, const hc::AmPointerInfo &srcPtrInfo, const hc::AmPointerInfo &dstPtrInfo) {
+HSACopy::enqueueAsyncCopyCommand(Kalmar::HSAQueue* hsaQueue2, const Kalmar::HSADevice *copyDevice, const hc::AmPointerInfo &srcPtrInfo, const hc::AmPointerInfo &dstPtrInfo) {
```

Comment:
It seems that the HSAQueue being passed in has become redundant, could we remove it from the signature?

Other classes derived from KalmarAsyncOps such as HSABarrier has similar issues